### ISSUE TITLE
Add support for Debian 12 (Bookworm)

### ISSUE THREAD

**Issue by drebs** [voxpupuli/puppet-php#690]
## Affected Puppet, Ruby, OS and module versions/distributions

- Puppet: (not applicable)
- Ruby: (not applicable)
- Distribution: Debian 12 Bookworm
- Module version: `v8.2.0` and current `master`

## How to reproduce (e.g Puppet code you use)

On a Debian 12 (Bookworm):

```
sudo apt install -y puppet git
git clone https://github.com/voxpupuli/puppet-php.git php
git clone https://github.com/puppetlabs/puppetlabs-stdlib.git stdlib
puppet apply --modulepath ./ -e 'include php'
```

## What are you seeing

```
Error: Evaluation Error: Error while evaluating a Function Call, Unsupported Debian release: 12 (file: /tmp/php/manifests/globals.pp, line: 38, column: 20) on node localhost
```

## What behaviour did you expect instead

Puppet should install PHP 8.2.

## Output log

See above in the "What are you seeing" section.

## Any additional information you'd like to impart

No, just thank you for developing this module!

**Comment by drebs** [comment]
To workaround, add the following to the Bookworm node's hiera data:

```yaml
php::globals::php_version: '8.2'
```

**Comment by poloz-lab** [comment]
It seems resolved with #687 in the [v10.2.0 release](https://github.com/voxpupuli/puppet-php/tree/v10.2.0)
